app/src/main/res/values/strings.xml

```diff
@@ -287,30 +283,29 @@
 
     <!-- Notifications -->
     <string name="updated_matches_notification_title">
-        Woohoo! Beeline has new matches for you!
+        Psst! Beeline has new matches for you!
     </string>
     <string name="updated_matches_notification_text">
-        Once a day, Beeline runs an algorithm to pair the hotties of the world. The algorithm just
-        finished running. You\'ve got some hot peeps to check out!
+        Once a day, Beeline runs an algorithm to pair you beautiful people. The algorithm just
+        finished running. You\'ve been matched!
     </string>
 
     <string name="only_lost_matches_notification_title">
         You\'ve had some matches in Beeline. Now they are gone.
     </string>
     <string name="only_lost_matches_notification_text">
-        Once a day, Beeline runs an algorithm to pair the hotties of the world. The last person
-        just lost interest in you. But fret not. There are many things you can do. (1) Rank more
-        users to find the special one who\'s into you. (2) Text your matches faster. Once you sent
-        the first message, the match becomes locked. (3) This is the hardest yet most satisfying.
-        Become more attractive. Hit the gym. Get new clothes. Put a smile on your face by following
-        your passion.
+        Once a day, Beeline runs an algorithm to pair you beautiful people. The last person
```

Comment:
This one turned out to be way too long either. I'll try to get you a screenshot of how little space in the Android system display is.